app/models/git_hub_user.rb

```diff
@@ -15,8 +15,12 @@ def login
   # Public
   #
   def admin_organization_memberships
+    no_cache_headers = { 'Cache-Control' => 'no-cache, no-store' }
```

Comment:
> I assumed the problem was with the Octokit caching going on over in https://github.com/education/classroom/blob/master/config/initializers/octokit.rb#L2

It does deal with Octokit, the reason why I added a special header here is because most of the time we want `shared_cache: false` because requests against users and organization are private so normally they won't cache.

Is this not advisable?